friends/src/friends/FriendForm.js

```diff
@@ -0,0 +1,69 @@
+import React, {useState } from 'react';
+import axiosWithAuth from '../login/axiosWithAuth'
+
+const FriendForm = (props) => {
+
+    const [newFriend, setNewFriend] = useState({
+        name:'',
+        age: '',
+        email:'',
```

Comment:
it might be cleaner if you made a variable `emptyFriend` and then used that to setup state and also clear the form after posting.